packages/flutter/test/widgets/shadow_test.dart

```diff
@@ -33,7 +33,7 @@ void main() {
       matchesGoldenFile('shadow.BoxDecoration.enabled.png'),
     );
     debugDisableShadows = true;
-  }, skip: isBrowser);
+  });
```

Comment:
>[ +138 ms] Preparing to send command: {"imageFile":"/tmp/flutter_web_platform.SVNUSH/imageXIEJHM/image","key":"shadow.BoxDecoration.enabled.png","update":false}
[   +1 ms] <<< Skipping "shadow.BoxDecoration.enabled.png" test : Golden file testing is unavailable on LUCI and  some Cirrus shards.

This is from the `web_tests-5-linux` shard.  I think you will need to add the gold service account, etc. to the shard that is executing the tests you would like to go through Gold. It looks like they are still skipping. `.cirrus.yml` configures the shards that use Gold: https://github.com/flutter/flutter/blob/6b11f18bc0c5a53753d1dc92928ec8ba1bdc5f95/.cirrus.yml#L90-L93

Reply:
Updated `.cirrus.yml`. Another thing to note is that even when the test is running, it would still fail because in `flutter_goldens`, the "no baseline" case[1] will only be hit when it's a new test being added.

In this case, it's not a new test, but a new "platform" instead. Do you think it's ok for us to ignore the failure in this PR, or should I fix the part in `flutter_goldens` as well?

[1]: https://github.com/flutter/flutter/blob/master/packages/flutter_goldens/lib/flutter_goldens.dart#L513

Reply:
I don't think ignoring the failures will go well. 

Since this will be a huge change to many golden files, it may be worth it to wait on #44474, which is just waiting on https://chromium-review.googlesource.com/c/infra/infra/+/1933027. This will enable tryjobs. You'll be able to see all of the files you generated in pre-submit checks and bulk triage them all at once so you'll land on green. 

Otherwise, you would have to use the current workflow that will require an ignore rule to be put in place for every test until you can land and triage - which is a bit tedious as it is designed against making bulk changes.

Reply:
I noticed also that all of the golden file tests are now failing on that shard, does the `skip: isBrowser` not work for the other tests?

Reply:
I've only enabled one test file with `matchesGoldenFile` right now, so only one file should be failing. Do you see other failures?

The reason it is failing is because it wasn't considered as "baseline not found". I've updated the PR with new changes in `flutter_goldens.dart` to consider the cases here as "baseline not found" as well, so the tests should pass now.

PTAL, thanks.